easybuild/easyconfigs/l/Loki/Loki-0.1.7_change_prefix.patch

```diff
@@ -0,0 +1,9 @@
+diff -ru loki-0.1.7.orig/Makefile.common loki-0.1.7/Makefile.common
+--- loki-0.1.7.orig/Makefile.common	2007-05-28 22:26:26.000000000 -0400
++++ loki-0.1.7/Makefile.common	2016-04-21 18:20:54.280476472 -0400
+@@ -4,4 +4,4 @@
+ CXXWARNFLAGS := -Wall -Wold-style-cast -Wundef -Wsign-compare -Wconversion -Wpointer-arith -pedantic
+ CXXFLAGS := $(CXXWARNFLAGS) -g -O2
+ 
+-prefix := /usr
++prefix := $(EB_INSTALL_DIR)
```

Comment:
I don't think this patch is needed? Have you tried using this?

```python
installopts = "prefix=%(installdir)s"
```

or

```python
preinstallopts = "export prefix=%(installdir)s && "
```

I think that should override the `/usr`, since they're using `:=` rather than `=`?